### ISSUE TITLE
Web part not loading 

### ISSUE THREAD

**Issue by terriyaki0212** [CloudDesignBox/cdb-promoted-links#8]
I encountered this error when adding the webpart to a modern page. Please advise is there something I'm missing? It's on the SiteAssets library of the site collection and my account has full control permissions to the library.

**[SPLoaderError.loadComponentError]:

***Failed to load component "5b0eac5a-50ba-4e01-8eb4-1767151d07f8" (CloudDesignBoxPromotedLinksWebPart).

Original error: ***Failed to load URL '/SiteAssets/CDBPromotedLinks/cdbpromotedlinks-clouddesignboxpromotedlinkswebpartstrings_en-us_765458216e451186e10982218c7e2d58.js' for resource 'CloudDesignBoxPromotedLinksWebPartStrings' in component '5b0eac5a-50ba-4e01-8eb4-1767151d07f8' (CloudDesignBoxPromotedLinksWebPart). There was a network problem.

***INNERERROR:

***Failed to load URL '/SiteAssets/CDBPromotedLinks/cdbpromotedlinks-clouddesignboxpromotedlinkswebpartstrings_en-us_765458216e451186e10982218c7e2d58.js' for resource 'CloudDesignBoxPromotedLinksWebPartStrings' in component '5b0eac5a-50ba-4e01-8eb4-1767151d07f8' (CloudDesignBoxPromotedLinksWebPart). There was a network problem.

***CALLSTACK:

Error
   at t (https://spoprod-a.akamaihd.net/files/sp-client-prod_2018-09-07.026/sp-pages-assembly_en-us_a4d1dcb466f9e5cc06396af0752db72f.js:745:15844)
   at t (https://spoprod-a.akamaihd.net/files/sp-client-prod_2018-09-07.026/sp-pages-assembly_en-us_a4d1dcb466f9e5cc06396af0752db72f.js:1605:21724)
   at e.buildErrorWithVerboseLog (https://spoprod-a.akamaihd.net/files/sp-client-prod_2018-09-07.026/sp-pages-assembly_en-us_a4d1dcb466f9e5cc06396af0752db72f.js:1605:15681)
   at e.buildLoadComponentError (https://spoprod-a.akamaihd.net/files/sp-client-prod_2018-09-07.026/sp-pages-assembly_en-us_a4d1dcb466f9e5cc06396af0752db72f.js:1605:11706)
   at Anonymous function (https://spoprod-a.akamaihd.net/files/sp-client-prod_2018-09-07.026/sp-pages-assembly_en-us_a4d1dcb466f9e5cc06396af0752db72f.js:1605:70119)**

**Comment by CloudDesignBox** [comment]
Please see this solution https://github.com/CloudDesignBox/cdb-promoted-links/issues/1
Needs to be in the site assets library of the root site collection.

**Comment by terriyaki0212** [comment]
Thank you so much, Tony! Yeah that resolves the error. I failed to check that it has to be at the root site collection (not just any site collection). I have added it now to a sample modern page. The tiles are okay but the background images are small and it doesn't occupy the entire tile like it used to on the classic page. How do I make the background images bigger? Is there a specific background image size that I need to use?
